source/fuzz/fact_manager.cpp

```diff
@@ -776,7 +792,7 @@ void FactManager::DataSynonymAndIdEquationFacts::
     ComputeCompositeDataSynonymFacts(const protobufs::DataDescriptor& dd1,
                                      const protobufs::DataDescriptor& dd2,
                                      opt::IRContext* context) {
-  // Check whether this is a synonym about composite objects.  If it is,
+  // Check whether this is a synonym about composite objects. If it is,
```

Comment:
https://www.cultofpedagogy.com/two-spaces-after-period/

(Though I am not quite 40 yet!)

Reply:
Nice :)